DCHdigi/README.md

```diff
@@ -0,0 +1,20 @@
+# Drift chamber (DCH) digitizers
+
+## DCHsimpleDigitizer
```

Comment:
this is the description of `DCHsimpleDigitizerExtendedEdm` can you change the "title", add that this version is meant to be used on DriftChamber_o1_v01 from k4geo and add the description of DCHsimpleDigitizer with something like "Algorithm for creating digitized drift chamber hits (based on edm4hep::TrackerHit3D) from edm4hep::SimTrackerHit. You have to specify the expected resolution in z and in xy (distance to the wire). The smearing is applied in the wire reference frame. Deprecated."